src/richie/plugins/lti_consumer/forms.py

```diff
@@ -0,0 +1,59 @@
+"""SimplePicture plugin forms."""
+from django import forms
+from django.utils.translation import gettext_lazy as _
+
+from .models import LtiConsumer
+
+
+class LtiConsumerForm(forms.ModelForm):
+    """Plugin form used to add a picture from the frontend admin."""
+
+    class Meta:
+        """Form meta attributes"""
+
+        model = LtiConsumer
+        fields = [
+            "lti_provider_id",
+            "url",
+            "oauth_consumer_key",
+            "shared_secret",
+        ]
+        widgets = {
+            "lti_provider_id": forms.Select(
+                choices=[
+                    (
+                        "custom",
+                        _("Choose a predefined LTI provider or fill fields below"),
+                    ),
+                    *LtiConsumer.available_providers(),
+                ],
+                attrs={"class": "form-control"},
+            ),
+        }
+
+    def clean(self):
+        if self.cleaned_data.get("lti_provider_id") == "custom":
```

Comment:
I think the custom LTI case should be when the `lti_provider_id` field is left blank.
```suggestion
        if self.cleaned_data.get("lti_provider_id"):
```

Reply:
I thought it would be clearer to have a "custom" value added to the field select, but leaving it empty is ok for me.